src/locales/en.json

```diff
@@ -135,5 +135,7 @@
 	"MODULES_NOT_DELETED": "Module are not deleted",
 	"MODULES_DELETED_SUCCESSFULLY": "Module deleted successfully",
 	"MODULES_HAS_EMPTY_LIST": "Module are empty for searchlist",
-	"MODULES_FETCHED_SUCCESSFULLY": "Module fetched successfully"
+	"MODULES_FETCHED_SUCCESSFULLY": "Module fetched successfully",
+	"PRIVATE_TYPE_NOT_ALLOWED": "Sorry, the specified type is not accessible for this session. Please check the type permissions.",
+	"CANNOT_EDIT_DELETE_SESSION_CREATED_BY_MANAGER": "Sorry, you are not authorized to edit or delete sessions created by the session manager."
```

Comment:
@nevil-mathew message can be - "Sorry, you are not authorized to edit or delete this session"